dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -109,9 +109,7 @@ def __init__(self, options: NumericalOptions = None) -> None:
                 "histogram": {"bin_counts": None, "bin_edges": None},
             }
         num_quantiles: int = 1000  # TODO: add to options
-        self.quantiles: list[float] | dict = {
-            bin_num: None for bin_num in range(num_quantiles - 1)
-        }
+        self.quantiles: list[float] = [bin_num for bin_num in range(num_quantiles - 1)]
```

Comment:
Modified quantiles to be set to list (previously could be list or dictionary which is ambiguous)

Reply:
I think this is giving false information.